src/model/Activity.java

```diff
@@ -0,0 +1,33 @@
+package model;
```

Comment:
Acho interessante um nome do pacote mais enxuto, mas melhor seguir a convenção do Java, do reversed domain name. Hoje temos o domain `.school`, por exemplo. Então, poderia ser:

```suggestion
package school.cicerone.model;
```